includes/namespace.php

```diff
@@ -80,6 +80,35 @@ function deactivate( $network_wide ) {
 register_activation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\activate' );
 register_deactivation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\deactivate' );
 
+if (
+	! class_exists( '\Google\Web_Stories\Plugin' ) ||
+	! file_exists( WEBSTORIES_PLUGIN_DIR_PATH . '/assets/js/edit-story.js' )
+) {
+	add_action(
```

Comment:
Need to print something when using WP-CLI too.

Reply:
I think we should follow this [pattern](https://github.com/xwp/wp-foo-bar/blob/6d60946c30a59bfa5df0cd7d415f77403d06cfd6/foo-bar.php#L35-L57). It is for php version numbers, but should work here.